src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1364,15 +1363,31 @@ IAssemblySymbol computeContainingAssembly(ISymbol s)
                         case SymbolKind.Discard:
                             s = ((IDiscardSymbol)s).Type;
                             continue;
+                        case SymbolKind.FunctionPointer:
+                            var funcPtr = (IFunctionPointerTypeSymbol)s;
+                            if (!isContainingAssemblyInReferences(funcPtr.Signature.ReturnType))
+                            {
+                                return false;
```

Comment:
>false [](start = 39, length = 5)

Is this code path covered by tests?

Reply:
Added a new test to `AccessCheckTests` for these.

---
In reply to: [426082638](https://github.com/dotnet/roslyn/pull/44257#discussion_r426082638) [](ancestors = 426082638)